modules/services/lorri.nix

```
@@ -0,0 +1,35 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+
+  cfg = config.services.lorri;
+
+in {
+
+  options = {
+    services.lorri = {
+      enable = mkOption {
+        type = types.bool;
+        default = false;
+        description = "Whether to enable to Lorri Daemon.";
+      };
+
+      package = mkOption {
+        type = types.path;
+        default = pkgs.lorri;
+        description = "This option specifies the lorri package to use.";
+      };
+    };
+  };
+
+  config = mkIf cfg.enable {
+    environment.systemPackages = [ cfg.package ];
+    launchd.user.agents.lorri = {
```

From what I recall the lorri daemon runs evaluation, which means in order to evaluate expressions with `builtins.fetchTarball` or `fetchGit` it will need access to tar/git. There's a `path` option for launchd services which provides a convenient way to expose binaries.

---

So I guess you can just add this:
```suggestion
    launchd.user.agents.lorri = {
    path = with pkgs; [ nix gitMinimal gnutar gzip ];
```